test/PushNotificationTests/MockRawNotification.cpp

```diff
@@ -0,0 +1,24 @@
+﻿#include "pch.h"
+#include "MockRawNotification.h"
+
+winrt::Windows::Foundation::Collections::IMapView<winrt::hstring, winrt::hstring> MockRawNotification::Headers()
+{
+    return winrt::Windows::Foundation::Collections::IMapView<winrt::hstring, winrt::hstring>{};
+}
+
+winrt::hstring MockRawNotification::ChannelId()
+{
+    return winrt::hstring(L"SampleChannelId");
+}
+
+winrt::hstring MockRawNotification::Content()
+{
+    return winrt::hstring(c_rawNotificationPayload);
```

Comment:
Definition of `c_rawNotificationPayload` in `inc\TestDef.h` should be marked `inline` rather than `static`. Otherwise, every file gets its own separate copy of `c_rawNotificationPayload`.